DOCS-2703: Add support for camera with esp32 (#3294)
viamrobotics/docs: docs/installation/_index.md

### Troubleshooting

> #### Error: IceTransportClosed

Reviewer: Ohhh nice, I've been definitely seeing that

If you are trying to connect to an ESP32 and the connection is unstable with the repeating error `E (412486) micro_rdk::common::webrtc::ice: closing ice agent with error IceTransportClosed`, you have opened too many connections to the ESP32, which has a maximum of 3 connections by default.

> Make sure you only have one tab open on your development machine that displays your machine details page for the machine on which your ESP32 is connected.

Reviewer: The on your development machine is actually wrong, because accessing it from two machines would also cause the error. They can however have other tabs open I assume. Just possibly only one tab with the Viam app OR only one tab that is on that particular machine page. Can you clarify that?
```suggestion
Make sure you only have one tab with your machine's page open to limit the number of connections to your ESP32.
```

Author: specifically with your machines page open not app in general so this seems right
